src/gamefield.ts

```diff
@@ -0,0 +1,48 @@
+export class GameField {
+    symbol: string;
+    width: number;
+    height: number;
+    row: string[];
+    gameField!: string[][];
+
+    constructor(symbol: string, widht: number, height: number) {
+        this.symbol = symbol;
+        this.width = widht;
+        this.height = height;
+        this.row = new Array(this.width).fill(this.symbol);
+    }
+
+    updateGameField() {
+        this.gameField = [];
```

Comment:
has to be set in constructor

Reply:
I move it from constructor to this method, because in the other way the height will increase with each update, so it needs to be in update method and constructor both?

Reply:
You have to initialise game field in constructor instead of using not null "!" operator.

Reply:
Done